student.rb

```diff
@@ -1,12 +1,7 @@
-class Student < Person
-  attr_reader :classroom
+class Student
+  attr_accesor :name, :classroom
 
-  def initialize(name = 'Unknown', age = nil, parent_permission: true, classroom: nil)
-    super(name, age, parent_permission: parent_permission)
-    @classroom = classroom
-  end
-
-  def play_hooky
-    '¯\(ツ)/¯'
+  def initialize(name)
+    @name = name
   end
 end
```

Comment:
- [ ] Do we really need to delete these lines of code? Kindly re-add them as they were. We need them so we can be able to create a Student who inherits from the Person class.

- [ ] Kindly add the other end of the association with Classroom class.

  ```
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```